### PR TITLE
Fix: Vimeo video now plays in lightbox context

### PR DESCRIPTION
**Pull request by mugurelLupu**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [x] I have tested this code to the best of my abilities
- [x] I have added unittests to verify the code works as intended
- [x] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

**Review by louiswol94 — APPROVED** [review]

File: tests/jest/unit/core/vimeo-loader.test.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	describe( 'getURLRegex', () => {
+		it( 'should return the correct URL regex', () => {
+			expect( vimeoLoader.getURLRegex().toString() ).toBe( '/^(?:https?:\\/\\/)?(?:www|player\\.)?(?:vimeo\\.com\\/)?(?:video\\/|external\\/)?(\\d+)([^.?&#"\'>]?)/' );
+		} );
+	} );
+
+	describe( 'getAutoplayURL', () => {
+		it( 'should return the correct autoplay URL', () => {
+			const videoURL = 'https://vimeo.com/123456#t=90s';
+			const expectedURL = 'https://vimeo.com/123456#t=90s';
```
Comment (hein-obox) [inline tests/jest/unit/core/vimeo-loader.test.js:29]: These `const` have the same value.

File: tests/jest/unit/core/vimeo-loader.test.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+			expect( vimeoLoader.getAutoplayURL( videoURL ) ).toBe( expectedURL );
+		} );
+
+		it( 'should handle URLs without a time parameter', () => {
+			const videoURL = 'https://vimeo.com/123456#t=';
```
Comment (hein-obox) [inline tests/jest/unit/core/vimeo-loader.test.js:34]: This assertion seems to be same as the previous one.

File: tests/jest/unit/core/vimeo-loader.test.js (hunk 1 above)
Comment (louiswol94) [reply tests/jest/unit/core/vimeo-loader.test.js:29]: It reads easier imo :)

File: tests/jest/unit/core/vimeo-loader.test.js (hunk 2 above)
Comment (louiswol94) [reply tests/jest/unit/core/vimeo-loader.test.js:34]: It is different URLs

**Review by hein-obox — APPROVED** [review]

**Pull request merged**